File: v2/java_sample/payloads/Originals.java
package payloads;

import org.json.JSONObject;

public class Originals {
	private String name;
	private int id;
	
	public Originals(String name, int id)
	{
		this.name = name;
		this.id = id;
	}
	
	public JSONObject toJson() 
	{
		JSONObject obj = new JSONObject();

		obj.put("name", this.name);
		obj.put("id", this.id);
		return obj;
		
	}

}

File: v2/node_sample/payload/Originals.js

class Originals
{
    name = "";
    id = 0;

    constructor(id, name)
  {
    this.id = id;
    this.name = name;
  }

  to_json(){
  return {"id": this.id, "name": this.name}
  }
}

module.exports = Originals